src/app/login/Login.tsx

```diff
@@ -14,16 +14,20 @@ const loginHelpLink = 'https://github.com/DestinyItemManager/DIM/wiki/Accounts-a
 
 export default function Login() {
   const dispatch = useThunkDispatch();
-  const authorizationState = uuidv4();
-  localStorage.setItem('authorizationState', authorizationState);
+  const authorizationState = useMemo(() => uuidv4(), []);
```

Comment:
This might fix some of the cases where people get the "auth state didn't match" error (it could happen before if this component re-rendered after the login link had been clicked)